server/index.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+const port = 3000;
```

Comment:
I think Port 3000 is being used for the client-side server. I think we might need to all have different port numbers to differentiate? Something to consider :)

Reply:
That's true, I've updated to 4000 now, thanks!